src/components/bookmark/BookmarkList.tsx

```diff
@@ -21,13 +21,29 @@ const BookmarkList = ({
         {bookmarkData.map((bookmark) => {
           const {
             contentId,
-            content: { type, parent, title, hidden, thumbnail },
+            content: { type, parent, title, hidden, thumbnail, courses },
           } = bookmark;
-          if (type === 'video' && parent && !hidden) {
+          if ((type === 'video' || type === 'notion') && parent && !hidden) {
             const { id: folderId, courses } = parent;
             const courseId = courses[0].courseId;
             const videoUrl = `/courses/${courseId}/${folderId}/${contentId}`;
 
+            return (
+              <ContentCard
+                type={type}
+                key={contentId}
+                title={title}
+                image={thumbnail || ''}
+                onClick={() => {
+                  router.push(videoUrl);
+                }}
+                bookmark={bookmark}
+                contentId={contentId}
+              />
+            );
+          } else if (type === 'folder' && !parent) {
```

Comment:
why this else sir?  from what i can see you are returning the same thing

Reply:
videoUrl is diff in both the case. there was already an implementation for type == 'video'. I do not want to break that. plus control will not go inside if in case of folder because parent will be null.